tests/mock_resource/property_definition.tfmock.hcl

```diff
@@ -0,0 +1,5 @@
+mock_resource "opslevel_property_definition" {
```

Comment:
Forgot to add this [in my already merged property definition PR. ](https://github.com/OpsLevel/terraform-provider-opslevel/pull/280)